tmt-app/src/Components/MajorCareerList.jsx

```diff
@@ -19,15 +19,15 @@ function MajorCareerList() {
           Majors
         </Typography>
         <Fragment>
-          {MAJORS_LIST.map((item) => (
+          {Object.entries(MAJORS_LIST).map((item) => (
```

Comment:
Any preference over the other Object methods? Entries are great imo when you need access to both `[key, value]` as a pair 

Might be just a nit pick, but I feel it would be more clear if we avoided the usage of index here in the callback here. `item[1]` had me double check why we were accessing via index 

What're your thoughts?

Reply:
Yea I only tried keys and forgot that I could get from the values also. I will change it so that we do not have to use index.